lib/game.dart

```diff
@@ -74,6 +106,40 @@ abstract class BaseGame extends Game {
   /// List of deltas used in debug mode to calculate FPS
   final List<double> _dts = [];
 
+  bool _checkTapOverlap(Tapeable c, Offset o) {
+    final pointRect = Rect.fromLTWH(o.dx, o.dy, 1, 1);
+
+    return c.toRect().overlaps(pointRect);
```

Comment:
the contains did not work? can we create a minimal example to reproduce this? might be a bug in flutter!

Reply:
Worked now, I think I didn't understood the suggestion before.